saleor/webhook/deprecated_event_types.py

```diff
@@ -67,6 +54,8 @@ class WebhookEventType:
     PAYMENT_PROCESS = "payment_process"
 
     SHIPPING_LIST_METHODS_FOR_CHECKOUT = "shipping_list_methods_for_checkout"
+    CHECKOUT_FILTER_SHIPPING_METHODS = "checkout_filter_shipping_methods"
```

Comment:
Should we add it to `deprecated_event_types.py` we also adding it to `event_types.py`

Reply:
It's on 3.0 so I think we need to keep it in deprecated event types even though we are adding it to 3.1 only now